pkg/controller/bundlec/bundle_sync_task.go

```diff
@@ -296,11 +297,29 @@ func (st *bundleSyncTask) updateBundle() error {
 }
 
 func (st *bundleSyncTask) updateBundleStatus() error {
+	// ------ BACKWARDS COMPATIBILITY with existing Bundle CRD without /status. REMOVE ONCE NOT NEEDED ------
+	obj, exists, err := st.store.Get(apiext_v1b1.SchemeGroupVersion.WithKind("CustomResourceDefinition"), meta_v1.NamespaceNone, smith_v1.BundleResourceName)
```

Comment:
We definitely have perms for this?

Reply:
This is reading an object from an informer's cache. This is not network IO. We do have permissions to read CRDs because we react dynamically to them being added and removed.